runtimes/alibi-explain/tests/helpers/sk_model.py

```diff
@@ -0,0 +1,66 @@
+import os
+from pathlib import Path
+import joblib
+
+from sklearn.ensemble import GradientBoostingClassifier
+from alibi.datasets import fetch_adult
+
+from mlserver import MLModel
+from mlserver.codecs import NumpyCodec
+from mlserver.types import InferenceRequest, InferenceResponse
+
+
+_MODEL_PATH = Path(os.path.dirname(__file__)).parent / ".data" / "sk_income" / "model.joblib"
```

Comment:
If the model is quick to train, there's no need for caching. I think we did initially that for the pytorch model bc it took a while, but generally the approach is to train on every round if it's quick enough to simplify the fixtures.

Reply:
75ab2e4c35343f599e944a3f2816355af833c944 removed caching and moved training to fixture